tutorials/monitoring/setup/index.md

```diff
@@ -0,0 +1,208 @@
+
+Click the Add button. Your Monitoring instance will start and be listed on the main page of the ServiceControl Management Utility.
+
+![ServiceControl Management Utility - List instance - Monitoring instance](scmu-list-mon-instance.png)
+
+The listing for the Monitoring instance includes a URL link. This URL will be needed when installing ServicePulse below.
+
+NOTE: Creating the Monitoring instance will also create the monitoring queue if it did not already exist. By default this queue is called _Particular.Monitoring_. 
+
+
+
+### Install ServicePulse
+
+ServicePulse is a web application for production monitoring and recoverability. It connects to a Monitoring instance to display monitoring data and to a ServiceControl instance to display recoverability data.
+
+Download and run the latest ServicePulse installer. 
+
+NOTE: If you already have ServicePulse running on your machine you will need to uninstall it and download the latest version in order to be able to configure ServicePulse to connect to a ServiceControl Monitoring instance.
+
+On the ServicePulse Configuration screen ensure that Recoverability is enabled and enter the URL of the ServiceControl instance API. Check the box marked Monitoring and enter the URL of the Monitoring instance API. 
+
+![ServicePulse Installer - ServiceControl configuration](sp-installer-sc-config.png)
+
+You can find the URLs for each instance API in the ServiceControl Management Utility.
+
+![ServiceControl Management Utility - Instance List - Api Addresses Highlighted](scmu-list-api-addresses.png)
+
+Launch ServicePulse and navigate to the Monitoring tab.
+
+![ServicePulse - Monitoring Tab - Empty](sp-monitoring-empty.png)
+
+NOTE: If Monitoring was not enabled during installation, the monitoring tab will not be visible.
+
+
+## Configure NServiceBus endpoints
+
+In order to be monitored by the Particular Service Platform, each endpoint needs to be configured to send data to three different queues: error, audit, and monitoring.
+
+It is recommended to go through this entire process for a single endpoint and ensure that it is working before continuing on to the next endpoint.
+
+
+### Error
+
+Whenever an NServiceBus endpoint is unable to process a message (even after several attempts) it will forward the message to the error queue. Each environment should contain a single error queue and should not share an error queue with another environment.
+
+The error queue is created with a new ServiceControl instance. By default, it is named `error`.
+
+Configure your endpoint to send failed messages to the error queue.
+
+partial: error-config
+
+NOTE: If you are using the MSMQ transport then you also need to specify the machine where the error queue is located. i.e. `error@MACHINENAME`. This should be the same machine where your ServiceControl instance is installed.
```

Comment:
I was assuming (most?) people will follow this tutorial after other tutorials, so they'll be using Learning transport. Would everything still make sense here? Should we use it as our "sample system" to monitor and for taking screenshots, etc.?
Or maybe use on-premise showcase as a sample?

Reply:
Unfortunately, we tried to get everything working with the Learning transport and ran in to issues. Neither ServiceControl instance type can be configured to use the learning transport at the moment.

We either need to fix that so that the platform tools can run on the Learning Transport or have an interim tutorial that describes picking a transport. Both are orthogonal to this page at the moment. Remember that this page is about adding monitoring to an existing system. It's targeted at existing customers.

Reply:
Ah I get it, fair enough. I'll add a note somewhere to make it explicit.